resources/authgear/templates/en/web/authflowv2/__password_field.html

```diff
@@ -23,9 +33,9 @@
     "PasswordRules" .PasswordRules
     "InputAttrs" $input_attrs
   ) }}
-  {{ if $has_err }}
+  {{ if $.HasError }}
```

Comment:
Checks ErrorMessage instead of HasError. So technically, it can display an error message without being highlighted in red.